client/src/Modal/Modal.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import "./Modal.css";
+export default function Mentors(props) {
+	const [isOpen,setIsOpen]=useState(true);
+
+    	return(
+    		<React.Fragment>
+    			{/* <button onClick={()=> this.setState({ isOpen:true })}>Open modal</button> */}
+    			{isOpen && (<div className='modal'>
+    				<div className='modal-body'>
+    					{/* <h1>{props.modalText}</h1> */}
+    					<h1>{props.modalText}</h1>
+					{props.func?<button onClick={()=> {
+						setIsOpen(false);
+						props.func();
+					}
+					}>proceed</button>:null}
```

Comment:
What's the difference between `proceed` and `ok`? `ok` seems to close the dialog without triggering the function, but `proceed` fires the function as well? If that's the case, `ok` should probably be `close` and `proceed` might be better as `ok` or `continue`.